blockchain/types/transaction.go

```diff
@@ -292,6 +292,14 @@ func (tx *Transaction) EffectiveGasPrice(baseFee *big.Int) *big.Int {
 	return tx.GasPrice()
 }
 
+func (tx *Transaction) AccessList() AccessList {
+	if tx.Type() == TxTypeEthereumAccessList || tx.Type() == TxTypeEthereumDynamicFee {
+		te := tx.GetTxInternalData().(TxInternalDataEthTyped)
+		return te.GetAccessList()
+	}
+	return []AccessTuple{}
```

Comment:
Wouldn't it be better to return nil?

Reply:
Looks like it is called only where accessList has to be existed.
Or can remove this function and type casting to access AccessList